src/patterns/definitions.owl

```diff
@@ -27964,12 +27986,12 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/OBA_0004272> "stomach acidity"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_0004272> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0001842> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/UBERON_0000945>)))
 
-# Class: <http://purl.obolibrary.org/obo/OBA_0004273> (cell aging onset quality)
+# Class: <http://purl.obolibrary.org/obo/OBA_0004273> (http://purl.obolibrary.org/obo/GO_0090398 onset quality)
```

Comment:
You have to refresh the import and then run the patterns again to get the correct label.